TGC/localisation/00_decisions.csv

```diff
@@ -1220,6 +1220,8 @@ first_cherokee_mounted_rifles_desc;Confederate officials commissioned Stand Wati
 first_cherokee_mounted_rifles_title;Form the First Cherokee Mounted Rifles;;;;;;;;;;;;;x
 flores_sale_desc;The Indonesian islands are almost worthless to us, but might be of value to the East India Company of another country. We should consider selling them for a tidy profit.;;;;;;;;;;;;;x
 flores_sale_title;The Flores Sale;;;;;;;;;;;;;x
+force_austria_to_join_KUK_desc;Now that Austria is in our sphere of influence it is time to force a greater answer to the German Question.;;;;;;;;;;;;;x
+force_austria_to_join_KUK_title;Force the German Question;;;;;;;;;;;;;x
```

Comment:
You don't have the latest hammonia changes in your branch, causing the dfference in your branches to include "rollbacks" for changes done in hammonia.

Reply:
I dont really know how to adress this. Is what is marked here everything that is changed? If so which are the rollbacks?